zenoss_hipchat/hipchat.py

```diff
@@ -88,27 +88,26 @@ def _should_notify(self):
         Determine if we should set the notify flag in HipChat call
         """
         if self.severity >= self.NOTIFY_SEVERITY:
-            return 1
-        return 0
+            return True
+        return False
 
     def send(self):
         """
         Send off the message to environment room with a nice template
         """
-
+        print(self.post_url)
         if not self.clear:
             message = self._event_message()
         else:
             message = self._clear_message()
 
-        from_name = "{0} ({1})".format(
+        from_name = "{1}".format(
             config.HIPCHAT_FROM, self.SEVERITY_MAP[self.severity][1]
```

Comment:
Actually, why change this at all?  You can always configure the FROM address yourself if you don't like seeing it come from `Zenoss`?  You could also add some logic to say:
```python
if config.HIPCHAT_FROM == '':
    from_name = self.SEVERITY_MAP[self.severity][1]
else:
    from_name = "{0} ({1})".format(
        config.HIPCHAT_FROM, self.SEVERITY_MAP[self.severity][1]
    )
```

I'd be fine with that